website/pages/ar/deploying/deploying-a-subgraph-to-hosted.mdx

```diff
@@ -284,8 +284,13 @@ Developers will be notified by email if one of their subgraphs has been marked a
 
 ## Subgraph Studio subgraph archive policy
 
-When a new version of a subgraph is deployed, the previous version is archived (deleted from the graph-node DB). This only happens if the previous version is not published to The Graph's decentralized network.
+A subgraph version in Studio is archived if and only if it meets the following criteria:
```

Comment:
This change looks like it's in the `ar` folder rather than the `en`?

Reply:
Wow nice catch, I missed that haha.